content/200-concepts/100-components/02-prisma-client/037-relation-queries.mdx

```diff
@@ -706,7 +633,7 @@ The following query creates ([`create`](/reference/api-reference/prisma-client-r
 <CodeWithResult>
 <cmd>
 
-```ts
+```ts highlight=4-6;normal
```

Comment:
the title of this section is "Connect an existing record" but this connects a new record with multiple existing records at once, maybe adjust the title?